<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_filme_2</class>
 <widget class="QMainWindow" name="tela_filme_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1121</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 900 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QFrame" name="inicial_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 31, 31);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="btn_Cine_filme">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 900 22pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>CineFilmes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Recomendacao_aleatorio">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Recomendação aleatório</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Inicio">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
color: rgb(255, 0, 0);
background-color: rgb(34, 31, 31);</string>
      </property>
      <property name="text">
       <string>Inicio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Filmes_Series">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
color: rgb(255, 0, 0);
background-color: rgb(34, 31, 31);</string>
      </property>
      <property name="text">
       <string>Filmes e Séries</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="btn_Ainda_Estou_Aqui">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmedois.qrc">:/icon 1/Ainda_Estou_Aqui.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Ainda_Estou_Aqui">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Ainda Estou Aqui</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_7_de_Novembro_2024_3h_15_minutos_Drma_Suspense">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string> 7 de Novembro de 2024 I 2h 15 minutos I Drama, Suspense</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Walter_Salles_Roteiro_Murilo_Hauser_Heitor_Lorenga">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção: Walter Salles I Roteiro: Murilo Hauser, Heitor Lorenga</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Fernanda_Torres_Fernanda_Montenegro_Selton_Mello">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Fernanda Torres, Fernanda Montenegro,  Selton Mello</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Ainda_Estou_aqui">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;O filme Ainda Estou Aqui conta a história de Eunice Paiva, que lutou por décadas para saber o paradeiro do marido, morto pela ditadura militar brasileira. O filme é baseado no livro homônimo de Marcelo Rubens Paiva, filho de Eunice e Rubens Paiva. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="btn_Vingadores_Ultimato">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>91</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmedois.qrc">:/icon 2/avengers.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Vingadores_Ultimato">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vingadores: Ultimato</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_25_de_Abril_2019_3h_01_minutos_Aventura_Acao_Ficcao_Cientifica">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>110</y>
       <width>411</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>25 de Abril de 2019 I 3h 01 minutos I Aventura, Ação, Ficção Científica</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Joe_Russo_Russo_Roteiro_Chistopher_Markus_Stephen_Mcfeely">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>130</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção: Joe Russo I Roteiro: Chistopher Markus, Stephen McFeely</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Robert_Downey_Jr_Chris_Evans_Mark_Ruffalo">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>150</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco:Robert Downey Jr, Chris Evans,Mark Ruffalo</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Vingadores_Ultimato">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>200</y>
       <width>491</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;br /&gt;Após Thanos eliminar metade das criaturas vivas, os Vingadores têm de lidar com a perda de amigos e entes queridos. Com Tony Stark vagando perdido no espaço sem água e comida, Steve Rogers e Natasha Romanov lideram a resistência contra o titã louco.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="btn_Diverdida_mente_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmedois.qrc">:/icon 3/disney.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Diverdida_mente_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>330</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diverdida Mente 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_20_de_Junho_de_2024_1h_36_minutos_Animacao_Comedia">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>20 de Junho de 2024 I 1h 36 Minutos I Animação, Comédia</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Kelsey_Man_Roteiro_Meg_LevFauve_Dave_Holstein">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>380</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção: Kelsey Man I Roteiro: Meg LeFauve, Dave Holstein</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Mia_Mello_Amy_Poehler_Isabella_Guarnieri">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Miá Mello, Amy Poehler, Isabella Guarnieri</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Diverdida_Mente_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>501</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Com o tempo, Riley cresce e enfrenta a adolescência. A sala de controle também se adapta, recebendo novas emoções. Alegria, Raiva, Medo, Nojinho e Tristeza ficam confusas com a chegada desses novos sentimentos.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="btn_Duna_Parte_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>310</y>
       <width>81</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmedois.qrc">:/icon 4/duna 2.webp</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Duna_parte2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>310</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duna: Parte 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_29_de_Fevereiro_de_2024_2h_46mintos_Drama_Ficcao_Cientifica">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>330</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>29 de Fevereiro de 2024 I 2h 46 minutos I Drama, Ficção Científica </string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Dennis_Villeneuve_Roteiro_Denis_Vileneuve_Jon_Spaihts">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>360</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção:Denis Villeneuve I Roteiro: Denis Villeneuve, Jon Spaihts</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Timothee_Chalamet_Zendeya_Rebecca_Ferguson">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>380</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Timothée Chalamet, Zendaya, Rebecca Ferguson</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Diverdida_Mente_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>440</y>
       <width>491</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;br /&gt;Paul Atreides se une a Chani e aos Fremen enquanto busca vingança contra os conspiradores que destruíram sua família. Enfrentando uma escolha entre o amor de sua vida e o destino do universo, ele deve evitar um futuro terrível que só ele pode prever.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_01">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>580</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>01</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_02">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>580</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>02</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_03">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>580</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>03</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_04">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>580</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>04</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_05">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>580</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>05</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../qrc/filmedois.qrc"/>
 </resources>
 <connections/>
</ui>
